test/planDecoder/testPlanDecoder.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/mongodb/atlas-osb/pkg/broker/dynamicplans"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	args := os.Args
+	if len(args) != 2 {
+		panic(errors.New("you need to pass a JSON file path as a second argument"))
+	}
+
+	path := args[1]
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		panic(fmt.Errorf("failed to read file %s: %w", path, err))
+	}
+
+	planEnc := string(data)
+	if []rune(planEnc)[0] == '{' {
+		planEnc = base64.StdEncoding.EncodeToString(data)
+	}
+
+	plan, err := decodePlan(planEnc)
+	if err != nil {
+		panic(fmt.Errorf("err: %w", err))
+	}
+
+	// fmt.Println(plan)
```

Comment:
rm comment

Reply:
I thought not to merge `testPlanDecoder.go` at all. Do we need it in the repo?

Reply:
I think it might be handy in the future if we encounter a similar problem